application/src/test/java/com/sanction/lightning/resources/FacebookResourceTest.java

```diff
@@ -197,38 +212,60 @@ public void testPublishWithNullFacebookResponse() {
     byte[] testBytes =  {};
     when(urlService.inputStreamToByteArray(inputStream)).thenReturn(testBytes);
     when(facebookService.publishToFacebook(any(byte[].class),
-            any(String.class), any(String.class))).thenReturn(null);
-    Response response = resource.publish(key, "Test", inputStream, contentDisposition, "Test");
+            any(String.class), any(String.class), any(String.class),
+            any(String.class))).thenReturn(null);
+    Response response = resource.publish(key, "Test", inputStream, contentDisposition, "photo",
+            "Test", "Test");
 
     assertEquals(response.getStatusInfo(), Response.Status.INTERNAL_SERVER_ERROR);
   }
 
   @Test
+  public void testPublishWithNullVideoTitle() {
+    byte[] testBytes =  {};
+    String testString = "Test";
```

Comment:
You don't need this local variable, you can just put the String literal `"Test"` in the `thenReturn()` method on line 230.